charts/blackduck-connector/values.yaml

```diff
@@ -0,0 +1,153 @@
+# Default values for Opssight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Docker registry to pull Black Duck images
+registry: docker.io/blackducksoftware
+imageTag: 2.2.5
+
+# image pull secret to download the Black Duck images (mostly applicable for air gapped customers)
+imagePullSecrets: [] # array of strings delimited by comma
+
+# This is used to start or stop the app. Set to Running to start, or Stopped to stop.
+status: Running
+
+logLevel: "debug"
+
+# External Hub ~>
+#  {
+#   "<BLACK_DUCK_URL>": 
+#     {
+#         "scheme":"https",
+#         "domain":"<BLACK_DUCK_URL>",
+#         "port":443,
+#         "user":"<BLACK_DUCK_USERNAME>",
+#         "password":"<BLACK_DUCK_PASSWORD>",
+#         "concurrentScanLimit":3
+#     }
+#   }
+
+# Secured Registries ~> 
+# {
+#   "<EXTERNAL_REGISTRY_URL>": 
+#   {
+#       "Url": "<EXTERNAL_REGISTRY_URL>",
+#       "user": "<EXTERNAL_REGISTRY_USERNAME>",
+#       "password": "<EXTERNAL_REGISTRY_PASSWORD>",
+#       "token": ""
+#   }
+# }
+
+secret:
+  externalHub: {}
+  securedRegistries: {}
+
+blackduck:
+  connectionsEnvironmentVariableName: "blackduck.json"
+  tlsVerification: false
+
+prometheus:
+  enabled: true
+  registry: docker.io/prom # override the docker registry at container level
+  imageTag: v2.1.0
+  expose: "None" #[None|LoadBalancer|NodePort|OpenShift]
+  resources:
+    requests:
+      cpu: 300m
+      memory: 1300Mi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  podSecurityContext: {}
+  securityContext: {}
+
+core:
+  registry:
+  imageTag:
+  port: 3001
+  timings:
+    checkForStalledScansPauseHours: 999999
+    stalledScanClientTimeoutHours: 999999
+    modelMetricsPauseSeconds: 15
+    unknownImagePauseMilliseconds: 15000
+    clientTimeoutMilliseconds: 100000
+  useMockMode: false
+  expose: "None" #[None|LoadBalancer|NodePort|OpenShift]
+  resources:
+    requests:
+      cpu: 300m
+      memory: 1300Mi
+
+processor:
+  port: 3002
+  certificate: ""
+  certificateKey: ""
+  annotationIntervalSeconds: 30
+  dumpIntervalMinutes: 30
+
+podProcessor:
+  enabled: true
+  registry:
+  imageTag:
+  nameSpaceFilter: ""
+  resources:
+    requests:
+      cpu: 300m
+      memory: 1300Mi
+
+scanner:
+  registry:
+  imageTag:
+  port: 3003
+  imageDirectory: "/var/images"
+  blackDuckClientTimeoutSeconds: 600
+  replicas: 1
+  resources:
+    requests:
+      cpu: 300m
+      memory: 1300Mi
+
+imageGetter:
+  registry:
+  imageTag:
+  port: 3004
+  host: "localhost"
+  imagePullerType: "docker"
```

Comment:
Is the default imagePullerType is Docker in Operator? Can you pls double check, I thought it is skopeo.

Reply:
I think you're right, it's skopeo